2333/2/versus.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'libs/options_parser.rb'
```

Comment:
It's inconvenient when you store `ParametersParser` class in `options_parser.rb` file. Use case: I want to open a file with `ParametersParser`, I press Cmd + P (try it in your editor ;-)) and type `pp` but there is no file like this.